ui.go

```diff
@@ -804,6 +804,24 @@ func (ui *ui) drawPromptLine(nav *nav) {
 	ui.promptWin.print(ui.screen, 0, 0, st, prompt)
 }
 
+func addStatLineVar(ruler []string, name string) []string {
+	if !strings.HasPrefix(name, "lf_") {
+		return ruler
+	}
+
+	val := os.Getenv(name)
+	if val == "" {
+		return ruler
+	}
+
+	if strings.HasPrefix(name, "lf_user_") {
+		return append(ruler, val)
+	} else {
+		val = strings.ReplaceAll(val, "\033", `\033`)
```

Comment:
Actually the reason why I added this in was because I realised that if the user tries to display options containing escape sequences like `tagfmt` then the ruler would be messed up.

I thought about this a bit more and I ended up making the following changes:

- `"\033[7m\\033\033[0m"` is now used to display escapes. I have no idea why a user would want to display the value of `tagfmt` or similar in the ruler, but maybe it's nice to display the exact string from the configuration. This is purely cosmetic and I suppose it can be changed at some later point in time if users prefer something else.
- The escaping is now done for user options as well. I actually intended to allow users to style values in this way, but I  guess it's also kind of strange if users are allowed to customise user options but none of the builtin values. I don't think any user has requested to *style* the values in the ruler, it was more just simply be able to display them. It's probably best if we ignore any styling and tackle it later if it becomes necessary.